src/main/java/pairmatching/controller/PairMatchingController.java

```diff
@@ -0,0 +1,185 @@
+package pairmatching.controller;
+
+import java.util.function.Function;
+import org.mockito.internal.util.Supplier;
+import pairmatching.enums.Course;
+import pairmatching.enums.Crew;
+import pairmatching.enums.Functions;
+import pairmatching.enums.Level;
+import pairmatching.enums.Missions;
+import pairmatching.enums.ShortAnswer;
+import pairmatching.model.MatchingRecord;
+import pairmatching.model.Pairs;
+import pairmatching.view.InputView;
+import pairmatching.view.OutputView;
+import pairmatching.view.error.ErrorException;
+
+public class PairMatchingController {
+
+    private static final String COMMA = "\\s*,\\s*";
+    private static final String EXIT = "exit";
+    public static final int MAX_TRY_COUNT = 3;
+
+    private InputView inputView;
+    private OutputView outputView;
+    private Crew backend;
+    private Crew frontend;
+    private MatchingRecord backendRecord;
+    private MatchingRecord frontendRecord;
+
+
+    public PairMatchingController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.backend = new Crew(Course.BACKEND);
+        this.frontend = new Crew(Course.FRONTEND);
+        this.backendRecord = new MatchingRecord();
+        this.frontendRecord = new MatchingRecord();
+    }
+
+    public void matchingSystemRun() {
+        while (true) {
+            try {
+                String input = getValidInput(inputView::displayStartMessage, Functions::getValidFunction);
+                executeWithInput(input);
+            } catch (IllegalArgumentException e) {
+                if (e.getMessage().equals(EXIT)) {
+                    return;
+                }
+                System.out.println(e.getMessage());
+            }
+        }
+    }
+
+    private void executeWithInput(String input) throws IllegalArgumentException {
+        if (Functions.isEquals(input, Functions.QUIT)) {
+            performFunctions(input, inputView);
+        }
```

Comment:
```suggestion
        if (Functions.isEquals(input, Functions.QUIT)) {
            performFunctions(input, inputView);
            return
        }
```
사소한거지만 return을 사용한다면 아래에 if문을 안만들고 처리할 수 있었을 거 같아요